frontend/components/PerceivedDemographics.jsx

```diff
@@ -75,18 +92,20 @@ class PerceivedDemographicsContainer extends Component {
             </select>
           </label>
 
-          <input type="submit" value="Submit" />
+          <button type="submit">Submit</button>
         </form>
-        <CarouselContainer/>
+        <CarouselContainer />
       </div>
-    )
+    );
   }
 }
 
-const mapStateToProps = state => ({});
+PerceivedDemographicsContainer.propTypes = {
+  onEnter: PropTypes.func.isRequired,
+};
```

Comment:
PropTypes allows us to enforce the values that are being passed to our component are of the proper type